src/Type.cpp

```diff
@@ -1911,7 +1926,11 @@ void OutputStructUnion(Type* type, std::ostream &out)
 			OutputUnionAssignOps(type, out, true);
 		}
 
-        out << "};";
+        out << "}";
+	if(type->eType == eStruct || type->eType == eUnion){
```

Comment:
Please check for coding style. Similarly for the other commits what have so far. Are you using a `clang-format` in the project?

Reply:
I am coding in a style as Csmith does. Like e.g
`t->used && (t->eType == eStruct || t->eType == eUnion)`

I am not sure whether Csmith codebase uses `clang-format` or not.

Reply:
Ok, then I'm fine with that.